### PR TITLE
feat: add quick start index code snippet

### PR DESCRIPTION
**Pull request by aradwann**
## Check List

- [x] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

related to #20

### CODE REVIEW

File: src/components/QuickStart/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import TabItem from '@theme/TabItem'
+import CodeBlock from '@theme/CodeBlock'
+
+export default function QuickStart({ esm, cjs }) {
+  return (
+    <>
+      <h1>Quick start</h1>
+      <p>Get fastify with NPM:</p>
+
+      <CodeBlock language="bash">npm install fastify</CodeBlock>
```
Comment (Eomm) [inline src/components/QuickStart/index.jsx:12]: I would add this too:
https://docusaurus.io/docs/2.2.0/markdown-features/code-blocks#npm2yarn-remark-plugin

File: src/pages/index.jsx [hunk 2]
```diff
@@ -83,8 +84,48 @@ export default function Home() {
 
         <section className="section">
           <div className="container">
-            <h1>Quick start</h1>
-            <p>TODO</p>
+            <QuickStart
+              esm={`
+// Require the framework and instantiate it
```
Comment (Eomm) [inline src/pages/index.jsx:89]:
```suggestion
              esm={`// Import the framework and instantiate it
```

File: src/pages/index.jsx [hunk 3]
```diff
@@ -83,8 +84,48 @@ export default function Home() {
...
+// Run the server!
+const start = async () => {
+  try {
+    await fastify.listen({ port: 3000 })
+  } catch (err) {
+    fastify.log.error(err)
+    process.exit(1)
+  }
+}
+start()
```
Comment (Eomm) [inline src/pages/index.jsx:109]:
```suggestion
// Run the server!
try {
  await fastify.listen({ port: 3000 })
} catch (err) {
  fastify.log.error(err)
  process.exit(1)
}
```

File: src/pages/index.jsx [hunk 4]
```diff
@@ -83,8 +84,48 @@ export default function Home() {
...
+            `}
+              cjs={`
+// Require the framework and instantiate it
```
Comment (Eomm) [inline src/pages/index.jsx:112]:
```suggestion
              cjs={`// Require the framework and instantiate it
```

**Review by Eomm — COMMENTED** [review]
Wow! nice work!

few extra suggestions

File: src/components/QuickStart/index.jsx (hunk 1 above)
Comment (aradwann) [reply src/components/QuickStart/index.jsx:12]: I will look into it

Comment (aradwann) [reply src/components/QuickStart/index.jsx:12]: npm2yarn plugin in docusaurus works with markdown, I think it doesn't work CodeBlock react component

Comment (Eomm) [reply src/components/QuickStart/index.jsx:12]: Let's add it to the "nice to have" features

**Review by Eomm — APPROVED** [review]

**Pull request merged**